agasc/scripts/mag_estimate_report.py

```diff
@@ -7,39 +7,56 @@
 import os
 import argparse
 import numpy as np
+import pathlib
 from astropy import table, time, units as u
-from cxotime import CxoTime
+from cxotime import CxoTime, units
 
 from agasc.supplement.magnitudes import mag_estimate_report
 
 
 def parser():
     parse = argparse.ArgumentParser(description=__doc__)
     parse.add_argument('--start', help='Time to start processing new observations.'
-                                       ' CxoTime-compatible time stamp.')
+                                       ' CxoTime-compatible time stamp.'
+                                       ' Default: now - 90 days')
     parse.add_argument('--stop', help='Time to stop processing new observations.'
-                                      ' CxoTime-compatible time stamp.')
-    parse.add_argument('--out-dir', default=f'./mag_stats_report',
-                       help='Output directory')
-    parse.add_argument('--obs-stats', default=f'mag_stats_obsid.fits',
-                       help='FITS file with mag-stats for all observations')
-    parse.add_argument('--agasc-stats', default=f'mag_stats_agasc.fits',
-                       help='FITS file with mag-stats for all observed AGASC stars')
-    parse.add_argument('--weekly-report', help="Add links to navigate weekly reports",
+                                      ' CxoTime-compatible time stamp.'
+                                      ' Default: now')
+    parse.add_argument('--input-dir', default='$SKA/data/agasc',
+                       help='Directory containing mag-stats files. Default: $SKA/data/agasc')
+    parse.add_argument('--output-dir', default='$SKA/www/ASPECT/agasc/supplement_reports/suspect',
```

Comment:
Still outputting to a flight directory.

Reply:
oops! It looks like I am actively trying to pass it under the rug...

Fixed it.